content/en/docs/contribution-guidelines.md

```diff
@@ -5,7 +5,7 @@ toc_hide: true
 ---
 
 OpenTelemetry is an open source project, and we gladly accept new contributions
-and contributors. Please see the CONTRIBUTING.md file in each SIG repository for
+and contributors. Please see the [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md) file in each SIG repository for
```

Comment:
Please use a relative path, our linters don't like absolute ones...

Reply:
This sentence is not referring to that particular file, but to all CONTRIBUTING.md files in all open-telemetry project repositories.

Reply:
Thank you for the additional content. I had the same doubts when I first watched it, so I proposed this change.